src/main/java/mate/jdbc/service/DriverService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
```

Comment:
The service must return an object of the class
```suggestion
    Driver get(Long id);
```